modules/swagger-codegen/src/main/resources/swift/Models.mustache

```diff
@@ -11,19 +11,8 @@ protocol JSONEncodable {
 }
 
 public enum ErrorResponse : ErrorType {
-    case RawError(Int, NSData?, ErrorType)
-    {{#operations}}
-        xxx
-    {{#operation}}
-        yyy
-        {{{operationId}}}
-    {{#responses}}{{^isDefault}}
-        zzz
-    case {{{operationId}}}{{{code}}}(Error)
-    {{/isDefault}}
-    {{/responses}}
-    {{/operation}}
-    {{/operations}}
+    case RawError(Int, NSData?, ErrorType){{#apiInfo.apis}}{{#operations}}{{#operation}}{{#responses}}{{^isDefault}}
+    case {{{operationId}}}{{{code}}}({{{dataType}}}){{/isDefault}}{{/responses}}{{/operation}}{{/operations}}{{/apiInfo.apis}}
```

Comment:
I wouldn't do this ... why not make this `ErrorResponse<T>` like `Response<T>` below?